cloudsmith_cli/cli/commands/auth.py

```diff
@@ -0,0 +1,58 @@
+"""CLI/Commands - Authenticate the user."""
+import webbrowser
+
+import click
+
+from .. import decorators, validators
+from ..exceptions import handle_api_exceptions
+from ..saml import get_idp_url
+from ..webserver import AuthenticationWebRequestHandler, AuthenticationWebServer
+from .main import main
+
+
+@main.command(aliases=["auth"])
+@click.option(
+    "-o",
+    "--owner",
+    metavar="OWNER",
+    required=True,
+    callback=validators.validate_owner,
+    prompt=True,
+    help="The name of the Cloudsmith organization to authenticate with.",
+)
+@decorators.common_cli_config_options
+@decorators.common_cli_output_options
+@decorators.initialise_api
+@click.pass_context
+def authenticate(ctx, opts, owner):
+    """Authenticate to Cloudsmith using the org's SAML setup."""
+    owner = owner[0]
+    api_host = opts.api_config.host
+
+    click.echo(
+        "Beginning authentication for the {owner} org ... ".format(
+            owner=click.style(owner, bold=True)
+        )
+    )
```

Comment:
NAB: Would be nice if we allow for configuring the owner in a profile instead so we follow the same practice we already have and it aligns with how AWS works too.

Reply:
> if we allow for configuring the owner in a profile instead

Do you mean in the CLI profile configuration, or some other mechanism?